### PR TITLE
added @o-ponchik to the ZTM-job-board

### PR DESCRIPTION
**Pull request by o-ponchik**
I have added @o-ponchik to the ZTM-job-board @dga

### CODE REVIEW

File: Submissions/@o-ponchik.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Mariia Matviian",
```
Comment (ProsperousHeart) [inline Submissions/@o-ponchik.json:2]: For confirmation - is it Mariia or Maria?

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Please also remove the @ from file name

File: Submissions/@o-ponchik.json (hunk 1 above)
Comment (o-ponchik) [reply Submissions/@o-ponchik.json:2]: the correct one is Mariia.

**Review by ProsperousHeart — APPROVED** [review]
Approved!

**Pull request merged**